pyvista/plotting/plotter.py

```diff
@@ -1778,7 +1793,22 @@ def _make_render_window_current(self):
 
     @property
     def image(self):  # numpydoc ignore=RT01
-        """Return an image array of current render window."""
+        """Return an image array of current render window.
+
+        Returns
+        -------
+        numpy.ndarray
```

Comment:
@Arnav-2004 Thanks for this. We get `pyvista.pyvista_ndarray` in my mind.
```suggestion
        pyvista.pyvista_ndarray
```
You can check it using type function.
```python
>>> type(pl.image)
pyvista.core.pyvista_ndarray.pyvista_ndarray
```

Reply:
`Plotter.get_image_depth` also returns `pyvista.core.pyvista_ndarray.pyvista_ndarray` but has been documented as `numpy.ndarray`

```py
>>> type(zval)
<class 'pyvista.core.pyvista_ndarray.pyvista_ndarray'>
```

Reply:
Same with `Plotter.screenshot`